packages/components/src/custom-gradient-picker/utils.js

```diff
@@ -75,7 +75,7 @@ export function getGradientAstWithControlPoints(
 			return {
 				length: {
 					type: '%',
-					value: position.toString(),
+					value: position?.toString(),
```

Comment:
This was throwing errors in Storybook due to `position` being undefined. To prevent those errors, I've added in optional chaining, which seemed to do the trick. It doesn't appear to change anything in real world usage in the editor, but made it easier to log out values while debugging in storybook.

Reply:
When will it error in storybook? Are there reproduction steps?

Reply:
It was erroring when the component is first loaded if you navigate to `?path=/story/components-customgradientpicker--default` and open up the console. I think in the story for this component, when it's initially mounted it doesn't yet have any position settings which caused the issue. We should probably also update the story, but making this function a little more tolerant to empty values was a quick win 🙂